client/asset/btc/btc_test.go

```diff
@@ -1259,79 +1311,88 @@ func TestFindRedemption(t *testing.T) {
 	// Prepare the "blockchain"
 	inputs := []btcjson.Vin{makeRPCVin(otherTxid, 0, otherSpendScript)}
 	// Add the contract transaction. Put the pay-to-contract script at index 1.
-	blockHash, contractBlock := node.addRawTx(contractHeight, makeRawTx(contractTxid, []dex.Bytes{otherScript, pkScript}, inputs))
-	getTxRes := &GetTransactionResult{
-		BlockHash:  blockHash.String(),
-		BlockIndex: contractHeight,
-		Details: []*WalletTxDetails{
-			{
-				Address:  contractAddr.String(),
-				Category: TxCatSend,
-				Vout:     contractVout,
-			},
-		},
+	contractTx := makeRawTx(contractTxid, []dex.Bytes{otherScript, pkScript}, inputs)
+	blockHash, contractBlock := node.addRawTx(contractHeight, contractTx)
+	contractTx.BlockHash = blockHash.String()
+	contractTx.Blocktime = contractBlock.Time
+	node.mpVerboseTxs[contractTxid] = contractTx
+
+	// Begin find redemption.
+	findRedemptionResultCh, err := wallet.FindRedemption(coinID)
+	if err != nil {
+		t.Errorf("unexpected FindRedemption error: %v", err)
+	}
+	// Expect to NOT find redemption yet.
+	select {
+	case frr := <-findRedemptionResultCh:
+		t.Errorf("unexpected FindRedemption result %v", frr)
+	case <-time.After(2 * time.Second):
```

Comment:
I reduced this to 1 second and though it passed for me locally, the tests failed CI. Commit https://github.com/decred/dcrdex/pull/513/commits/486d77c593f5fc48a9cb4b151139785d2c6b6cf0, CI results https://github.com/decred/dcrdex/runs/1030219180.

It appears 1 second may not be enough time to find a redemption in this test, and it's important to wait sufficiently to prevent false positives: a scenario where we expect the redemption to NOT be found, and it was found within 2 seconds but not within 1 second. The test would pass when it should have failed.